mobilecoind-json/src/bin/main.rs

```diff
@@ -295,12 +295,8 @@ fn create_request_code(
     // Generate b58 code
```

Comment:
Confirming: the changes in this file are all improving syntax/style, but not material to mixed token types?

Reply:
Yes, there was a lot of discussion about u64s in Json earlier, that ended with, the JsonU64 thing is fine if we use in other places